wamr-compiler/CMakeLists.txt

```diff
@@ -398,4 +396,4 @@ else()
                          ${UV_A_LIBS})
 endif()
 
-install (TARGETS wamrc DESTINATION bin)
+install (TARGETS wamrc)
```

Comment:
Was this done on purpose? Others might need it.

Reply:
Yes, I changed it to fix issues during packaging in Yocto. Since CMake 3.14 (which is already set as the minimum required version in this CMake file), the install command automatically adds DESTINATION ${CMAKE_INSTALL_BINDIR} if not specified. Including GNUInstallDirs ensures that CMAKE_INSTALL_BINDIR is set correctly, with a fallback to bin. Since this is a cache variable, users can override it to meet specific needs, which is often necessary for packaging.